src/components/ha-selector/ha-selector-state.ts

```diff
@@ -0,0 +1,49 @@
+import { html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import { StateSelector } from "../../data/selector";
+import { SubscribeMixin } from "../../mixins/subscribe-mixin";
+import { HomeAssistant } from "../../types";
+import "../entity/ha-entity-state-picker";
+
+@customElement("ha-selector-state")
+export class HaSelectorState extends SubscribeMixin(LitElement) {
+  @property() public hass!: HomeAssistant;
+
+  @property() public selector!: StateSelector;
+
+  @property() public value?: any;
+
+  @property() public label?: string;
+
+  @property() public helper?: string;
+
+  @property({ type: Boolean }) public disabled = false;
+
+  @property({ type: Boolean }) public required = true;
+
+  @property() public context?: {
+    filter_entity?: string;
+  };
+
+  protected render() {
+    return html`
+      <ha-entity-state-picker
+        .hass=${this.hass}
+        .entityId=${this.selector.state.entity_id ||
+        this.context?.filter_entity}
+        .value=${this.value}
+        .label=${this.label}
```

Comment:
Let's add a default label that translates to "State"

Reply:
It has: <https://github.com/home-assistant/frontend/pull/13411/files#diff-81f9f49d4ef814fb9b2857609d79bf419ee2064fcd9011fb077229331b2a71daR79-R80>

On level up :)

I did notice I forgot a transition, I will add that.